libfreerdp/core/transport.c

```diff
@@ -371,6 +371,37 @@ BOOL transport_accept_nla(rdpTransport* transport)
 	return TRUE;
 }
 
+void transport_bio_error_log(char *biofunc, rdpTransport* transport)
```

Comment:
In this file could you do some early returns so that we don't have these cascading if() ?